usds/justice40-tool
# Re-order nav links, update download box to include shapefiles, squishy pages

# Purpose
Re-order the navigation links. This PR creates all the placeholder logic to add dropdown navigation links in the future. The TSD, FAQs and download page are also ready to go when we are ready in the future

- closes #1425 
- closes #1533 
- closes #1532
- closes #1436 
- closes #1324 

## QA testing round 2

### QA link [here](http://usds-geoplatform-justice40-website.s3-website-us-east-1.amazonaws.com/justice40-tool/1536-57b952/en/methodology/)

### QA spec:

Add shapefile download to download box
- [ ] does download button work?
- [ ] how is the styling of the box?
- [ ] how do colors look? Blue public engagement button near the darker blue download box?
- [ ] placement of the new Tag?

Add FAQ link to About page
- [ ] the text has a minimum 92 character requirement. that's why the copy has been updated. Is it ok, or do you have something else that could also meet the 92 character limit?

## QA testing round 1

<details>

### QA link [here](http://usds-geoplatform-justice40-website.s3-website-us-east-1.amazonaws.com/justice40-tool/1536-eb645c/en/#3/33.47/-97.5)

### QA spec:

Re-order the nav links:
- [x] does the map/explore page load by default?
- [x] do all the navigation links work?
- [x] do all (most) links work as expected on each page?

Engagement calendar button on each page
- [x] is there a public engagement button on each page?
- [x] does the button work as expected?

Add FAQ link to About page
- [x] does the FAQ link work?

Modify footer
- [x] is the title of the second column in the footer correct?
- [x] are all footer links in the second column correct?
- [x] are all the footer links in the second column in the correct order?
- [x] is the title of the third column in the footer correct?

Questions for QA:
1. ~We have 3 different buttons on the site. A blue with white text, white with dark text, and yellow with blue text. I have a feeling that having at most two buttons unless there's some reason to have 3 different types. Wondering if we can discuss the options and see if all 3 button types are needed or if they cause extra cognitive load?~
</details>
